internal/featuresupport/features.go

```diff
@@ -721,3 +721,41 @@ func (feature *MinimalIso) getFeatureActiveLevel(_ *common.Cluster, infraEnv *mo
 	}
 	return activeLevelNotActive
 }
+
+// MceFeature
+type MceFeature struct{}
+
+func (feature *MceFeature) New() SupportLevelFeature {
+	return &MceFeature{}
+}
+
+func (feature *MceFeature) getId() models.FeatureSupportLevelID {
+	return models.FeatureSupportLevelIDMCE
+}
+
+func (feature *MceFeature) getName() string {
+	return "multicluster engine"
+}
+
+func (feature *MceFeature) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
+	if !isFeatureCompatibleWithArchitecture(feature, filters.OpenshiftVersion, swag.StringValue(filters.CPUArchitecture)) {
+		return models.SupportLevelUnavailable
+	}
+
```

Comment:
Please validate the minimum OCP version here, return `models.SupportLevelUnavailable` if requirement doesn't met

Reply:
added minimum version